src/main/java/seedu/address/logic/commands/AddBuyerCommand.java

```diff
@@ -1,12 +1,12 @@
 package seedu.address.logic.commands;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_POSTALCODE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
```

Comment:
'seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL' should be before 'seedu.address.logic.parser.CliSyntax.PREFIX_PHONE'.